test/broadcast.jl

```diff
@@ -68,12 +63,12 @@ srand(42)
                     @dbg f,t,n1,n2
                     a1 = rand11(f,t,n1)
                     a2 = rand11(f,t,n2)+t(1)
-                    @test gradcheck(f1, Any[a1, a2])
+                    @test gradcheck(f1, [a1, a2])
```

Comment:
We sometimes force Any[] type so when AutoGrad tries to replace elements of an array with their ::Rec counterparts we don't get an error, because the outer array type only allows for numeric arrays and not Rec's etc.  This was not the case here?